F/FFMPEG/FFMPEG/build_tarballs.jl

```diff
@@ -23,22 +23,22 @@ products = [
 dependencies = [
     HostBuildDependency("YASM_jll"),
     BuildDependency("nv_codec_headers_jll"),
-    Dependency("libass_jll"; compat="0.15.1"),
+    Dependency("libass_jll"; compat="0.15.2"),
     Dependency("libfdk_aac_jll"; compat="2.0.3"),
     Dependency("FriBidi_jll"),
-    Dependency("FreeType2_jll"; compat="2.10.4"),
+    Dependency("FreeType2_jll"; compat="2.13.3"),
     Dependency("LAME_jll"),
     Dependency("libvorbis_jll"),
     Dependency("libaom_jll"),
     Dependency("Ogg_jll"),
     BuildDependency("LibVPX_jll"), # We use the static archive
     Dependency("x264_jll"; compat="10164.0.0"),
-    Dependency("x265_jll"; compat="~3.6"),
+    Dependency("x265_jll"; compat="4.1"),
```

Comment:
Hard to predict, but it's possible that x265 will break ABI in next minor version, because they did it before (reason why the previous compat was `~3.6` and not just `3.6`)

Reply:
Ok.